fix: clarify instructions for catphotoapp step 6 (#46985)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23991f86c76b9248c6eb8.md

# --description--

> In the previous step, you put the `h2`, comment, and `p` elements inside the `main` element, which is known as nesting. Nested elements should be placed two spaces further to the right of the element they are nested in. Such a spacing is called an indentation and is used to make HTML easier to read.

Reviewer: minor suggestion - could go either way, I've just been trying to make language simpler
```suggestion
In the previous step, you put the `h2`, comment, and `p` elements inside the `main` element. This is called *nesting*. Nested elements should be placed two spaces further to the right of the element they are nested in. This spacing is called indentation and it is used to make HTML easier to read.
```

Author: I originally has this as two sentences as well.
